server/CMakeLists.txt

```diff
@@ -36,6 +36,7 @@ option(TRITON_ENABLE_STATS "Include statistics collections in backend utilities"
 option(TRITON_BUILD_TEST "Include unit test for the Server Wrapper" ON)
 option(TRITON_ENABLE_EXAMPLES "Include examples in build" ON)
 
+option(TRITON_BUILD_STATIC_LIBRARY "Create multiple static libraries, otherwise create one dynamic library" ON)
```

Comment:
I was wondering what the other static libraries will be? Is the change to the build required by java or just some improvements?

Reply:
The other static libraries were created in conjunction to `tritondevelopertoolsserver`. I'm not sure what they are used for. I made no change to the static libraries

Reply:
I see. I thought we were building some new static libraries. Thanks for clarifying.